mlir/test/lib/Interfaces/TilingInterface/TestTilingInterfaceTransformOps.cpp

```diff
@@ -203,6 +204,12 @@ DiagnosedSilenceableFailure
 transform::TestFuseConsumerOp::apply(TransformRewriter &rewriter,
                                      TransformResults &transformResults,
                                      TransformState &state) {
+  SmallVector<Operation *> slices;
+  for (auto op : getTargets()) {
+    auto sliceOp = *state.getPayloadOps(op).begin();
+    slices.push_back(sliceOp);
```

Comment:
```suggestion
    slices.push_back(*state.getPayloadOps(op).begin());
```

Reply:
Id rather keep it separate. Its easier to read that way for me.